integration_test/firtool/registers.fir

```diff
@@ -0,0 +1,33 @@
+; REQUIRES: verilator
+; RUN: firtool %s -verilog -o %t1.sv
+; RUN: circt-rtl-sim.py %t1.sv --cycles 32 2>&1 | FileCheck %s
```

Comment:
It seems like verilator lint-only would be enough to test valid verilog.

Reply:
I'd still like to have at least a short test that simulates the output.